### ISSUE TITLE
Monero XMR/BTC Volume Booster Bounty 5000 $BSQ

### ISSUE THREAD

**Issue by flix1** [bisq-network/growth#176]
### Monero XMR/BTC Volume Booster Bounty

The goal of this bounty is to increase XMR/BTC trading volume.

To claim this bounty a contributor must outline and execute a strategy over a 3 month period to attract traders and increase volume traded in the XMR/BTC market by 25%.

https://bisq.network/markets/?currency=xmr_btc

1. Declare intention to claim bounty
2. Outline strategy
3. Report on progress and detail work done here

4. **IF** volume over the 3 month period following the start of contributors work is **25% higher** than the 3 months preceding **THEN** the contributor can claim the bounty.

**Comment by flix1** [comment]
![image](https://user-images.githubusercontent.com/4233784/72068725-066e9a00-32e6-11ea-9cd4-1b73d29fee97.png)

Considering how high XMR/BTC volumes already are on Bisq and how important this volume is to Bisq, I consider the reward, while high compared to other bounties, well worth paying out if the contributor is successful.

**Comment by flix1** [comment]
![image](https://user-images.githubusercontent.com/4233784/72146780-7429cd00-339d-11ea-9482-9e6dfd2a5a68.png)

As a reference  XMR/BTC volume for the past 3 months was:
October 2019         373 BTC
November 2019     375 BTC
December 2019     328 BTC

**Comment by chimp1984** [comment]
How is the acceptence process of a proposal? We need to make very clear which candidate/concept we will take as bounty candidate. I assume it only makes sense with one single candidate.

**Comment by flix1** [comment]
> We need to make very clear which candidate/concept we will take as bounty candidate. I assume it only makes sense with one single candidate.

I agree it only makes sense to accept one proposal. The idea is for the candidate to outline his strategy here and have a discussion... No candidate is going to put in the work and time before being sure that the DAO will pay up if he delivers required results.

**Comment by MwithM** [comment]
> How is the acceptence process of a proposal? We need to make very clear which candidate/concept we will take as bounty candidate. I assume it only makes sense with one single candidate.

For an already bootstrapped market like XMR is, I think that lowering fees could improve volume more than a bounty.
All market participants could benefit from lower fees and the "reward" would be automatic and transparent.

There's an economic behaviour consequence to consider here, though. People tends to prefer rewards over lower fees even if the rational decision would be to prefer the other because we enjoy a reward (considered as a kind of gift) while we consider lower fees as a lower punishment.

The best option would be to reward those market participants who can prove they managed to do a certain volume of trading, but I know it's not possible considering Bisq privacy needs.

I wanted to discuss the possibility of just lowering fees for XMR, and maybe a mix of low bounties and low fees for other low liquid assets.

**Comment by flix1** [comment]
@MwithM I agree that lowering fees is the best long term policy to attract and retain volume, together with adapting the software to the specific market needs.

However some initial marketing effort is needed. We don't have anybody in Bisq doing it... so offering a bounty to reward somebody who is well placed to do it could be a good option. 

It costs nothing if it fails. If it succeeds it creates a new Bisq contributor out of someone capable to attract other traders and who can teach us. After learning to use Bisq, earning BSQ... they normally stick around and continue to help the project.

Basically I consider the bounty primarily a recruiting tool, not just a marketing tool.

**Comment by flix1** [comment]
Very interesting feedback from Monero community here:
https://www.reddit.com/r/Monero/comments/eo2ssy/monero_bounty_on_bisq/
https://bisq.community/t/dirty-btc-coins-on-the-xmr-market/7798

2 takeaways:
-XMR traders worried about tainted BTC
-Traders value high execution speed

**Comment by flix1** [comment]
> Well, for me the bounty doesn't read as if they have improvements on the technical side on the top of their minds here, but more things like better targeting of marketing.

Let me clarify: technical improvements _are_ part of the strategy to increase volume. For example in the JPY/BTC bounty #138 the work involved translating to Japanese and customizing for local payment methods.

The point of linking the bounty to measurable results is so that we don't waste anybody's time. Technical changes must be highly demanded by actual users. A bounty hunter must outline a viable strategy to reach the targets outlined in the bounty.

And who better to know what is needed than an actual XMR trader? For example  the bounty hunter could say "IF X improvement was implemented, that would convince me to use Bisq and I in turn could convince many of my fellow traders". Then a conversation would start with the devs to get X done and once completed the bounty could be claimed IF the bounty target is achieved.

**Comment by MwithM** [comment]
>Takeaways:
>1. XMR traders worried about tainted BTC
>2. Centralized exchanges blocking accounts
>3. Traders value high execution speed

1. To worry about tainted BTC is a XMR maximalist proposition that none exchange can solve just like Bisq can't solve bank payment privacy issues. I don't feel BTC is dead, though. You always can use Wasabi or Wirlpool to mix your BTC after buying BTC on Bisq (and the worries of tainted BTC don't apply to selling BTC for XMR).
2. Cryptobanks are going to behave like fiat banks.
3. I guess that if the minimum limit for trade deposit and other measures makes arbitration cases reduced, the 10 days limit to send funds for arbitration could be lowered considerably. And there's the instant altcoin payment method for 1 hour limit trades. All markets could improve with higher execution speeds.

There's another point against this bounties, at least when the objective is to promote a market that already has some volume. How do you ask for bounty hunter's identity? I imagine we need to do it to stop me or any other contributor or an already active trader from participating.

**Comment by flix1** [comment]
> How do you ask for bounty hunter's identity? I imagine we need to do it to stop me or any other contributor or an already active trader from participating.

We don't ask for identity. We don't care about identity. Bounty only cares about outcomes. In fact if somebody wants to create a pseudonymous github user to claim the bounty while maintaining their privacy... that is perfect.

I don't see the problem with an already active trader participating, since this bounty is calculated to pay for itself in fees, any existing contributor is welcome to take a crack at it. They would just be wasting everyones time, since we want sustainable growth.  In that scenario (existing trader just trades with himself to fake volume) the 5000 $BSQ could end up being worth less than the 0.4% paid in fees for those trades.

**Comment by MwithM** [comment]
I was asking because if one of us or an already active traders takes this challenge the promotional aspect will be smaller, I'm not worried about self trading.

**Comment by MwithM** [comment]
I was asking because if one of us or an already active traders takes this challenge the promotional aspect would be smaller, I'm not worried about self trading.

**Comment by flix1** [comment]
Bounty closed as unclaimed.
